<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite thread-count="1" parallel="tests" name="Failed suite [Suite]" guice-stage="DEVELOPMENT" verbose="0">
  <test thread-count="1" parallel="tests" name="Test6(failed)" verbose="0">
    <classes>
      <class name="Runner_Files.Headerpage_positive_scenarios_runner">
        <methods>
          <include name="runScenario" invocation-numbers="4 5 6 7 8 9 10"/>
          <include name="tearDownClass"/>
          <include name="setUpClass"/>
        </methods>
      </class> <!-- Runner_Files.Headerpage_positive_scenarios_runner -->
    </classes>
  </test> <!-- Test6(failed) -->
  <test thread-count="1" parallel="tests" name="Test7(failed)" verbose="0">
    <classes>
      <class name="Runner_Files.Help_positive_scenarios_runner">
        <methods>
          <include name="runScenario" invocation-numbers="1 4"/>
          <include name="tearDownClass"/>
          <include name="setUpClass"/>
        </methods>
      </class> <!-- Runner_Files.Help_positive_scenarios_runner -->
    </classes>
  </test> <!-- Test7(failed) -->
  <test thread-count="1" parallel="tests" name="Test11(failed)" verbose="0">
    <classes>
      <class name="Runner_Files.Request_second_opinion_negative_scenarios_runner">
        <methods>
          <include name="runScenario" invocation-numbers="15"/>
          <include name="tearDownClass"/>
          <include name="setUpClass"/>
        </methods>
      </class> <!-- Runner_Files.Request_second_opinion_negative_scenarios_runner -->
    </classes>
  </test> <!-- Test11(failed) -->
  <test thread-count="1" parallel="tests" name="Test22(failed)" verbose="0">
    <classes>
      <class name="Runner_Files.Award_points_positive_scenarios_runner">
        <methods>
          <include name="runScenario" invocation-numbers="2"/>
          <include name="tearDownClass"/>
          <include name="setUpClass"/>
        </methods>
      </class> <!-- Runner_Files.Award_points_positive_scenarios_runner -->
    </classes>
  </test> <!-- Test22(failed) -->
  <test thread-count="1" parallel="tests" name="Test23(failed)" verbose="0">
    <classes>
      <class name="Runner_Files.Forgot_password_negative_scenarios_runner">
        <methods>
          <include name="runScenario" invocation-numbers="0 1 2 4"/>
          <include name="tearDownClass"/>
          <include name="setUpClass"/>
        </methods>
      </class> <!-- Runner_Files.Forgot_password_negative_scenarios_runner -->
    </classes>
  </test> <!-- Test23(failed) -->
  <test thread-count="1" parallel="tests" name="Test24(failed)" verbose="0">
    <classes>
      <class name="Runner_Files.Forgot_password_positive_scenarios_runner">
        <methods>
          <include name="runScenario" invocation-numbers="1 2"/>
          <include name="tearDownClass"/>
          <include name="setUpClass"/>
        </methods>
      </class> <!-- Runner_Files.Forgot_password_positive_scenarios_runner -->
    </classes>
  </test> <!-- Test24(failed) -->
</suite> <!-- Failed suite [Suite] -->
